src/components/Action/Action.vue

```diff
@@ -82,6 +87,9 @@ export default {
 		},
 		firstAction() {
 			return this.actions[0]
+		},
+		moreIconClass() {
+			return `icon-more${this.forceWhite ? '-white' : ''}`
```

Comment:
How do we make sure this is also white on the dark theme then?

Reply:
Good question! This is still an unsolved issue, right?

Reply:
For this one we could just ship the white more icon with the lib, since it will always be rendered on dark background.

Reply:
@juliushaertl shall we do the same with the close icon then?

Reply:
Yes, at least until we have a proper other solution, I'd just include them if they are not needed to be inverted.